LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+Copyright (c) 2019 Aldwin Vlasblom
```

Comment:
Should users be prompted to update this line?

Reply:
Yes. Does sanctuary-scripts do/check this based on `author-name`?

Reply:
Yes, it does. <https://github.com/sanctuary-js/sanctuary-scripts/blob/v3.2.0/bin/update-copyright-year#L9>:

```bash
author="$(get author-name)"
```